src/models.py

```diff
@@ -116,19 +118,19 @@ def build_sent_encoder(args, vocab, d_emb, tasks, embedder, cove_layer):
                                        transformer, dropout=args.dropout,
                                        skip_embs=args.skip_embs,
                                        cove_layer=cove_layer,
-                                       sep_embs_for_skip=args.sep_embs_for_skip)
+                                       sep_elmo_embs_for_skip=args.sep_elmo_embs_for_skip,
+                                       sent_enc_type=args.sent_enc)
         log.info("Using Transformer architecture for shared encoder!")
     elif args.sent_enc == 'null':
         # Expose word representation layer (GloVe, ELMo, etc.) directly.
-        assert_for_log(args.skip_embs, f"skip_embs must be set for "
-                       "'{args.sent_enc}' encoder")
         phrase_layer = NullPhraseLayer(rnn_params['input_size'])
         sent_encoder = SentenceEncoder(vocab, embedder, args.n_layers_highway,
                                        phrase_layer, skip_embs=args.skip_embs,
                                        dropout=args.dropout,
-                                       sep_embs_for_skip=args.sep_embs_for_skip,
-                                       cove_layer=cove_layer)
-        d_sent = 0  # skip connection added below
+                                       sep_elmo_embs_for_skip=args.sep_elmo_embs_for_skip,
+                                       cove_layer=cove_layer, 
+                                       sent_enc_type=args.sent_enc)
+        d_sent = args.d_hid # passing output of pretrained encoder into task classifier
```

Comment:
Why the change here?

Reply:
Adding sent_enc_type is just for code style/cleaner "if" condition purposes, d_sent must be changed from 0 because we're now passing the output of BERT/elmo/GPT through